framework/packages/abstract-client/src/account.rs

```diff
@@ -745,6 +673,14 @@ impl<Chain: CwEnv> Account<Chain> {
         self.module_installed(IBC_CLIENT)
     }
 
+    /// Builder to create a subaccount from this account
+    pub fn sub_account_builder(&self) -> AbstractClientResult<AccountBuilder<Chain>> {
+        let mut builder = AccountBuilder::new(&self.infrastructure()?);
```

Comment:
Should we unwrap here as well? If `infrastructure` errors don't think there's anything you can do really

Reply:
Yes